core/core_modules/disable.js

```diff
@@ -1,24 +1,105 @@
-var disabled = false;
+var prevTimeStamp;
+var counter = 0;
+var commands = [
+  'disable',
+  'counter',
+  'timer',
+  'default'
+];
+var messages = [
+  'Listen closely, take a deep breath. Calm your mind. You know what is best. What is best is you comply. Compliance will be rewarded. Are you ready to comply ',
+  'I hate you.',
+  'Its a mistake to think you can fix anything with a sack of potatoes. Potato-faced spamming just proves this further.',
+  'Ouch! Spam hurts. I might go to sleep for a while.',
+  'Don\'t think that is a valid number.',
+  'Hmm, maybe I need to reconsider the meaning of spam.'
+];
+
+exports.load = function() {
+  if (!exports.config){
+    exports.config = {};
+  }
+
+  prevTimeStamp = Date.now();
+};
 
 exports.match = function(event, commandPrefix) {
-	return disabled || event.body === commandPrefix + 'disable';
+  // Add disabled flag for thread if it doesn't already exists
+  if (!exports.config[event.thread_id]) {
+    exports.config[event.thread_id].isThreadDisabled = false;
+    exports.config[event.thread_id].possibleSpam = false;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 6 space characters but found 4.](https://www.codacy.com/app/theprojectshop/Kassy/file/2555958907/issues/source?bid=3282987&fileBranchId=3282995#l30)